plugins/woocommerce/src/Internal/Admin/Orders/Edit.php

```diff
@@ -215,9 +234,37 @@ public function handle_order_update() {
 		// Order updated message.
 		$this->message = 1;
 
-		// Refresh the order from DB.
-		$this->order = wc_get_order( $this->order->get_id() );
-		$theorder    = $this->order;
+		$this->redirect_order( $this->order );
+	}
+
+	/**
+	 * Helper method to redirect to order edit page.
+	 *
```

Comment:
Same on `@since`

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/39193/commits/0629e2fb6389f8958f70c61cbb25901dd496179d